### PR TITLE
Adrv9009 mcs

### PR DESCRIPTION
**Pull request by dNechita**

### CODE REVIEW

File: plugins/adrv9009.c [hunk 1]
```diff
@@ -261,8 +261,8 @@ static void multichip_sync()
 
 		guint n = 0;
```
Comment (commodo) [inline plugins/adrv9009.c]: why not squash this into the previous patch ?

**Comment by commodo** [comment]
rest looks good

File: plugins/adrv9009.c (hunk 1 above)
Comment (dNechita) [reply plugins/adrv9009.c]: Because it is @mirceacaprioru's work and findings.

Comment (commodo) [reply plugins/adrv9009.c]: ok
fine from me

**Comment by commodo** [comment]
[LGTM](https://i-h2.pinimg.com/564x/d4/f2/00/d4f20041254a0727ddce7cb81be9e68c.jpg?b=t)

**Pull request merged**